openapi/components/schemas/user-readableFields.yaml

```diff
@@ -7,11 +7,7 @@ properties:
     example: 24
 
   campus:
-    type: array
-    minItems: 1
-    uniqueItems: true
-    items:
-      $ref: ./campus.yaml
+    $ref: ./campus.yaml
```

Comment:
This was actually correct (a user has a list of campuses): the example in `openapi/components/schemas/campus.yaml` is what needs to be fixed.

The example in that file is a list which contains an object:
```yaml
example:
  - id: 1
    name: Secondary
    code: SEC
    days: 5
```

It should just be a single object:
```yaml
example:
    id: 1
    name: Secondary
    code: SEC
    days: 5
```